mempool/clist_mempool.go

```diff
@@ -96,6 +100,10 @@ func NewCListMempool(
 	return mp
 }
 
+func (mem *CListMempool) SetSwitch(sw *p2p.Switch) {
```

Comment:
The p2p switch only needed in the list mempool right? Are any other mempool that would need this? If so we could add to the interface but that might be breaking

Reply:
I wouldn't change the interface, as this is to me a hopefully temporary workaround.